lua/plenary/path.lua

```diff
@@ -198,6 +199,54 @@ function Path:expand()
   return expanded and expanded or error("Path not valid")
 end
 
+function Path:make_relative(filepath, cwd)
```

Comment:
Take a look how other functions handle parameters. No other function takes a filepath.
Thats because this is "basically" a class. We would call it like this:
```lua
Path:new(filepath):make_relative(cwd)
```

Take a look what `print/vim.inspect(self))` is or what `:new` does.

It would be also cool to be able to not pass cwd and if thats the case use the cwd of the object.

Reply:
That applies for all your methods :)